Troubleshooting.md

```diff
@@ -67,15 +68,17 @@ Or in case this occurs for a field in one of your classes which you did not actu
 
 **Reason:** You probably have not configured ProGuard / R8 correctly
 
-**Solution:** Make sure you have configured ProGuard / R8 correctly to preserve the names of your fields. See the [Android example](examples/android-proguard-example/README.md) for more information.
+**Solution:** Make sure you have configured ProGuard / R8 correctly to preserve the names of your fields.
+See the section below, it's related to this issue.
```

Comment:
Directly link to the section as well?
```suggestion
See the [ProGuard / R8](#r8--proguard) section for more information.
```

Reply:
This is intended. As I see it, the "random property names" and "field names are being obfuscated" are the same issue; I just wanted to link them together.

Reply:
We probably should merge these 2 sections.

Reply:
I would keep them separate for now. The symptoms are different (but similar) and if I recall correctly users had experienced and asked about both problems already in the past.